src/NServiceBus.RavenDB/Outbox/RavenDbOutboxStorage.cs

```diff
@@ -16,16 +16,24 @@ public RavenDbOutboxStorage()
 
         protected override void Setup(FeatureConfigurationContext context)
         {
-            var store = DocumentStoreManager.GetDocumentStore<StorageType.Outbox>(context.Settings);
             var endpointName = context.Settings.EndpointName();
 
-            Helpers.SafelyCreateIndex(store, new OutboxRecordsIndex());
+            DocumentStoreManager.GetUninitializedDocumentStore<StorageType.Outbox>(context.Settings)
```

Comment:
Could we make `SafelyCreateIndex` a bit more precise e.g. `CreateOnStartup`. 

Can we move the method to `DocumentStoreManager`? It's not really clear why we need to call `GetUninitializedDocumentStore` and what uninitialized means here.

Reply:
happy to rename 👍 
I guess we could wrap this inside the DocumentStoreManager, but it would still do the exact same thing?

Reply:
> I guess we could wrap this inside the DocumentStoreManager, but it would still do the exact same thing?

Yes that was my thinking as well.

Reply:
I renamed the method but I decided to not add the same method to the store manager. I think it's good enough for the two invocations for now and with the new name, the behavior should be easier to understand.